apps/passport-server/src/services/telegramService.ts

```diff
@@ -812,6 +819,43 @@ export class TelegramService {
       );
     }
 
+    if (!nullifierHash) throw new Error(`Nullifier hash not found`);
```

Comment:
we also need to be checking that the claim.externalNullifier matches the expected externalNullifier

Reply:
see https://github.com/proofcarryingdata/zupass/blob/main/apps/consumer-client/src/pages/examples/zk-eddsa-event-ticket-proof.tsx#L467-L468